content/integrations/rollbar.md

```diff
@@ -3,6 +3,7 @@ title: Datadog-Rollbar Integration
 integration_title: Rollbar
 kind: integration
 doclevel: basic
+description: "Send exceptions, errors, and code deployments to your Datadog event stream."
```

Comment:
Send exceptions, errors, and code deployments from Rollbar to your Datadog event stream.

Reply:
👍